src/headset/oculus.c

```diff
@@ -449,6 +449,11 @@ static void oculusRenderTo(void (*callback)(void*), void* userdata) {
   }
 }
 
+static Texture* oculusGetMirrorTexture() {
+  ovr_GetMirrorTextureBufferGL(state.session, state.mirror, &handle);
```

Comment:
I am confused, where is `handle` declared?

Reply:
That looks like an error. I don't think this should compile.
My test did compile against the oculusvr sdk, so that tells me my test must have been testing the wrong thing.
I can't test again today but tomorrow I should be able to do a more rigorous test.